admin_guide/manage_nodes.adoc

```diff
@@ -315,67 +315,67 @@ xref:marking-nodes-as-unschedulable-or-schedulable[marked unschedulable] to
 perform pod evacuation.
```

Comment:
@adellape Doesn't let me comment on line 313, but maybe remove the "or nodes" here too?

Reply:
+1